test/test_SW_SoilWater.cc

```diff
@@ -53,82 +50,71 @@ namespace{
   TEST(SWSoilWaterTest, SWSWCSdjustSnow){
```

Comment:
There can be a few more unit tests for this function:
*  temp_ave > TminAccu2;
* What do you expect snowmelt to be based on various if else in  snow melt section. You should test at least 2 conditions, if not more.